Plugins/GitUIPluginInterfaces/IGitPlugin.cs

```diff
@@ -28,4 +28,34 @@ public interface IGitPlugin
         /// </summary>
         bool Execute(GitUIEventArgs args);
     }
-}
\ No newline at end of file
+
+    public interface IGitPluginV2
```

Comment:
How many plugins are maintained independently of the GE repo? Could we drop supporting the initial interface?
I guess most of them are still there. (I'll celebrate the day they will be removed from the standard build.)